modules/server/src/network/resolvers/index.ts

```diff
@@ -1,8 +1,24 @@
-import { NetworkAggregationConfig } from '../types';
+import { NetworkAggregationConfig, NetworkFieldType } from '../types';
+import { createAggregationResolvers } from './aggregations';
 import { resolveRemoteConnectionNodes } from './remoteConnections';
 
-export const createResolvers = (networkConfigsWithSchemas: NetworkAggregationConfig[]) => ({
-	Query: {
-		nodes: async () => await resolveRemoteConnectionNodes(networkConfigsWithSchemas),
-	},
-});
+/**
+ * Create GQL resolvers
+ * @param networkConfigsWithSchemas
+ * @param networkFieldTypes
+ * @returns
+ */
+export const createResolvers = (
+	configs: NetworkAggregationConfig[],
+	networkFieldTypes: NetworkFieldType[],
+) => {
+	const aggregationsResolvers = createAggregationResolvers(configs, networkFieldTypes);
+
+	return {
+		Query: {
+			nodes: async () => await resolveRemoteConnectionNodes(configs),
+			aggregations: () => [], // TODO: Why need array returned? needs some truthy val?
```

Comment:
unclear about this and currently investigating. We don't return anything for just an `aggregations` field, but it does need to return something truthy so sub fields resolve correctly